lib/puppet_pal.rb

```diff
@@ -310,24 +310,6 @@ def self.evaluate_script_manifest(manifest_file)
     end
   end
 
-  # Runs the given named plan passing arguments by name in a hash.
-  # @param plan_name [String] the name of the plan to run
-  # @param plan_args [Hash] arguments to the plan - a map of plan parameter name to value, defaults to empty hash
-  # @param manifest_file [String] a Puppet Language file to load and evaluate before running the plan, mutually exclusive with code_string
-  # @param code_string [String] a Puppet Language source string to load and evaluate before running the plan, mutually exclusive with manifest_file
-  # @return [Object] returns what the evaluated plan returns
-  # @deprecated Use {#with_script_compiler} and then `call_function('run_plan', plan_name, plan_args)` on the given compiler - to be removed in 1.0 version
-  #
-  def self.run_plan(plan_name,
```

Comment:
ok to remove but bolt must be in sync and use call_function instead

Reply:
Yes, that's the plan.